src/main/resources/application.properties

```diff
@@ -188,10 +188,12 @@ max-pool-size=2
 queue-capacity=500
 
 ## everwell variables
-everwellUserName = <Enter everwell username>
-everwellPassword = <Enter everwell password>
-amritUserName = <Enter AMRIT username>
-amritPassword = <Enter AMRIT password>
+everwellUserName = Demo_API_SVA
+
+everwellPassword = Demo_API_SVA
+
+amritUserName = 1097user
+amritPassword = Test@123
```

Comment:
@swetatech24 Replace it with the placeholder. Do not put credentials in properties file.